.eslintrc.js

```diff
@@ -37,6 +39,7 @@ const config = {
       'error',
       {
         paths: [
+          ...baseConfig.rules['no-restricted-import'][1].paths,
```

Comment:
Being a bit defensive in case we change the base config
```suggestion
          ...(baseConfig?.rules['no-restricted-import']?.[1]?.paths ?? []),
```